### ISSUE TITLE
System-Prompt support

### ISSUE THREAD

**Issue by xuminwlt** [openpilot-hub/devpilot-intellij#11]
如果针对系统固化的提示词带来的回复不满意，可以提供额外的提示词进行更加自定义化的能力，实现更多元化的风格，当然也可以预置几种系统风格。

**Comment by xiangtianyu** [comment]
Good idea, maybe we will add github sso later and provide more customized prompt configurations in future.
